compiler-rt/lib/mpk_untrusted/mpk_fault_handler.cpp

```diff
@@ -90,7 +90,10 @@ void disableMPK(siginfo_t *si, void *arg) {
 void stepMPKHandle(int sig, siginfo_t *si, void *arg) {
   __sanitizer::Report(
       "Reached signal handler after single instruction step.\n");
-  enableThreadMPK(arg, si->si_pkey);
+  auto handler = __mpk_untrusted::AllocSiteHandler::get();
```

Comment:
Looking at the `get()` API here, it should probably be `GetOrInit()`, so that it is clear what is going on. This is unrelated to this PR, but would be good to fix.

Reply:
Changed naming to `getOrInit()` in accordance with comment.